README.rst

```diff
@@ -8,6 +8,12 @@ Installation
 ------------
 :code:`pip install hca`.
 
+Install the libmagic library.  On Linux machines this library is already present.
```

Comment:
I think you should not assume libmagic is installed. For example I'm not sure if it's installed in the Docker Ubuntu distribution by default. Suggest adding the apt-get and yum install lines.

Reply:
Also, I'm worried about depending on libmagic. In the past I've seen this and other native library dependencies be really troublesome. Suggest looking into using a pure Python implementation e.g. https://github.com/cdgriffith/puremagic.

Reply:
If this were the server code, I'd be okay with this.  However, end users will run into this and not everyone is comfortable with homebrew.

Reply:
I switched to puremagic.